prisma/seed.ts

```diff
@@ -6,29 +6,12 @@ import {
   getPlusSuggestionsData,
   getPlusStatusesData,
 } from "./mocks/plus";
-import { getUsersData } from "./mocks/user";
+import userFactory from "../utils/factories/user"
```

Comment:
I might prefer to keep the factories under the `prisma` folder as they are something we never need unless we are seeding data. So maybe the path could just be `./factories/user` here. What do you think? I'm not quite sure what is the best but having them close to the `seed.ts` file seems the best to me when taking the first look.